### PR TITLE
Add script 'udit'.

### PR DESCRIPTION
**Pull request by Uditgulati**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,3 +29,16 @@ This repo contains some useful bash scripts which are making my day-to-day life
 		usage example: bash ping_network.sh
 		after execute the script pass you network
 		eg: 192.168.0.1
+
+## 6.udit
```
Comment (vishichoudhary) [inline README.md:33]: Name of script is not informative

File: README.md [hunk 2]
```diff
@@ -29,3 +29,16 @@ This repo contains some useful bash scripts which are making my day-to-day life
...
+        A simple script that takes contest name as input and goes to that folder, removes previous files, creates new files using predefined template.
+        
+	Usage
+	
+        copy file udit to folder "/usr/local/bin/" and file1.cpp, file2.cpp to "~/Documents/Format/". Run command udit <Contest_Name>. Add these lines to .bashrc
```
Comment (vishichoudhary) [inline README.md:38]: Why to add /usr/local/bin you can add the directory in PATH variable

Comment (vishichoudhary) [reply README.md:38]: Instead of moving file file1.cpp file2.cpp you can use cvim template

File: README.md (hunk 1 above)
Comment (Uditgulati) [reply README.md:33]: Couldn't be more informative than this

File: README.md (hunk 2 above)
Comment (Uditgulati) [reply README.md:38]: Well, there are multiple ways, you gotta do it some way. IMO my way is easy to understand and easy to implement.

**Pull request merged**